fr.md

```diff
@@ -41,6 +41,7 @@ Sur Internet, il existe de nombreux tutoriels/cours pour apprendre le développe
 
 #### ![FR](https://raw.githubusercontent.com/learndev-info/awesome-learning-dev-fr/master/medias/franceflag.png)
 
+* [Guide complet](https://developer.mozilla.org/fr/docs/Apprendre/Commencer_avec_le_web)
```

Comment:
In the EN section and the EN version (en.md) you can add the english version of the link (https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web)